### PR TITLE
Update OWNERS: add CFE team

### PR DESCRIPTION
**Pull request by thejasn**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,10 +4,16 @@ reviewers:
   - sttts
   - deads2k
   - slaskawi
+  - thejasn
+  - TrilokGeer
+  - swghosh
```
Comment (xingxingxia) [inline OWNERS:9]: I like to be appended as optional reviewer : ) I'm OpenShift Auth QE lead who will also test cert-manager, like to review when I can. If not appropriate to be added, nvm that is fine for me.

Comment (thejasn) [reply OWNERS:9]: Will raise a separate PR adding you as reviewer.

**Pull request merged**